application/src/main/java/org/togetherjava/tjbot/features/componentids/InvalidComponentIdFormatException.java

```diff
@@ -14,7 +14,8 @@ public final class InvalidComponentIdFormatException extends RuntimeException {
     /**
      * Creates a new instance.
      */
-    public InvalidComponentIdFormatException() {}
+    public InvalidComponentIdFormatException() {
+    }
```

Comment:
i tried searching for it but can't find how to fix this @marko-radosavljevic pls provide the config entry to fix this. Or if there isn't any and all other changes are ok then we can merge this